Gemfile

```diff
@@ -94,6 +94,7 @@ group :test do
   gem 'rails-controller-testing'
   gem 'mocha'
   gem 'webmock'
+  gem "minitest-stub_any_instance"
```

Comment:
@alis-khadka no need to add this gem, we already have stubbing support via `mocha` (eg: test/controllers/simple_discussion/forum_threads_controller_test.rb)